MSCL/source/mscl/MicroStrain/MIP/MipTypes.h

```diff
@@ -1677,6 +1679,35 @@ namespace mscl
 
     typedef std::vector<DeviceCommPort> CommPortInfo;
 
+    //API Struct: EventTriggerInfo
+    //  Information about an event trigger
+    struct EventTriggerInfo
+    {
+        //API Enum: Status
+        //  Available trigger statuses
+        //      ACTIVE  - 0x00 - Trigger is active
+        //      ENABLED - 0x01 - Trigger is enabled
+        //      TEST    - 0x02 - Trigger is in test mode
+        enum Status
+        {
+            ACTIVE  = 0x00, // Trigger is active
+            ENABLED = 0x01, // Trigger is enabled
+            TEST    = 0x02  // Trigger is in test mode
+        };
+
+        //API Variable: type
+        //  Configured trigger type
+        uint8 type;
```

Comment:
This needs to be an EventTriggerConfiguration::Trigger value. Probably need to move this class to ExposedInertialTypes - likely makes sense for it to be there anyway, it's where the other trigger types are defined.